sdk/core/azure-core/src/main/java/com/azure/core/implementation/SwaggerMethodParser.java

```diff
@@ -53,6 +54,7 @@
  * method.
  */
 public class SwaggerMethodParser implements HttpResponseDecodeData {
+    private ClientLogger logger = new ClientLogger(SwaggerInterfaceParser.class);
```

Comment:
The argument to `ClientLogger` constructor should match the classname in which it's initialized.

```suggestion
    private ClientLogger logger = new ClientLogger(SwaggerMethodParser.class);
```